tmcprototype/new_configure_logging.py

```diff
@@ -0,0 +1,82 @@
+import sys
+import json
+import time
+import numpy as np
```

Comment:
Looks like this import statement is not used anywhere in the script. Remove it if not required. Check other import statements as well.

Reply:
Done.

Reply:
Commit ID: d8488c155e50d5c4940dbe52585ac2446c43334a